pyproject.toml

```diff
@@ -29,7 +29,7 @@ typing-extensions = ">=4.2.0"
 cachetools = "^4.2.2"
 types-cachetools = "^4.2.4"
 websockets = "^10.3"
-solders = "^0.10.0"
+solders = { git = "https://github.com/hubbleprotocol/solders.git", branch = "vali/versioned-txn" }
```

Comment:
cc @kevinheavey, can you take a look at https://github.com/kevinheavey/solders/pull/32

Reply:
Hey, thanks for looking at this! The combination of these two PRs is enough for VersionedTransaction to work (both legacy and V0). Unfortunately the code breaks the API (it should use a `union[Transaction, VersionedTransaction]`) and has some hacky parts as the interface between Message/VersionedMessage and Transaction/VersionedTransaction has changed a bit. But hopefully this serves as a baseline for starting the work on this feature. At the moment I don't really have capacity to make this nicer, it's mostly a solution for who really needs versioned transactions in solana right now.

Also I cannot figure out how to format the files corretly, have used `black` with line limit set to 120 but it ignores the pylint ignore comments and formats some dependencies weirdly.

Reply:
run ‘make format’ for formatting